nchs_mortality/tests/conftest.py

```diff
@@ -38,9 +59,7 @@ def run_as_module(date):
     with mock_s3():
```

Comment:
is it difficult to skip this by removing the "archive" key like we did in some of the other indicators?

Reply:
I think it is good to not omit this because the daily caching seems to be an integral part of this indicator and it probably won't be superseded by the archiver in the near future.